server/bin/create_version_file.js

```diff
@@ -17,10 +17,24 @@
  *
  */
 
+const fs = require('fs-extra');
 const path = require('path');
 
-module.exports = {
-  DIST_PATH: path.resolve(__dirname, 'server/dist'),
-  ROOT_PATH: path.resolve(__dirname),
-  SRC_PATH: path.resolve(__dirname, 'src'),
-};
+const distFolder = 'dist';
+
+function padTo2Digits(num) {
+  return num.toString().padStart(2, '0');
+}
+
+function formatDate(date) {
+  return `${[date.getFullYear(), padTo2Digits(date.getMonth() + 1), padTo2Digits(date.getDate())].join('.')}.${[
+    padTo2Digits(date.getHours()),
+    padTo2Digits(date.getMinutes()),
+  ].join('.')}`;
+}
+
+const version = formatDate(new Date());
```

Comment:
Why not using the previous technique?

```suggestion
const {format} = require('date-fns');
const version = format(new Date(), 'yyyy.MM.dd.HH.mm');
```